tools/RunVersionController.ps1

```diff
@@ -290,6 +291,21 @@ switch ($PSCmdlet.ParameterSetName)
             }
         }
 
+        foreach ($requiredModule in $localAz.RequiredModules)
+        {
+            $galleryDependency = $galleryAz.Dependencies | where { $_.Name -eq $requiredModule.Name }
+            if ($galleryDependency -eq $null)
+            {
+                Write-Host "New module $($requiredModule.Name) is found in local Az" -ForegroundColor Yellow
+                $updatedModules += $requiredModule.Name
+                if ($versionBump -eq [PSVersion]::NONE -or $versionBump -eq [PSVersion]::PATCH)
+                {
+                    $versionBump = [PSVersion]::MINOR
+                    Write-Host "Adjust $versionBump version bump" -ForegroundColor Yellow
```

Comment:
It's suggested to describe in more details, such as "Adjust $versionBump version bump for Az"

Reply:
Updated.